sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -16,12 +16,13 @@
 /**
  * Helper class to build CosmosAsyncClient {@link CosmosAsyncClient} and CosmosClient {@link CosmosClient}
  * instances as logical representation of the Azure Cosmos database service.
- *
+ * <p>
  * When building client, endpoint() and key() are mandatory APIs, without these the initialization will fail.
- *
+ * <p>
  * Though consistencyLevel is not mandatory, but we strongly suggest to pay attention to this API when building client.
- * By default, database account level consistency level is used if none is provided.
- *
+ * By default, SESSION consistency level is used if none is provided.
```

Comment:
```suggestion
 * By default, account consistency level is used if none is provided.
```

Reply:
This will enable us treat it a as bug fix for later, otherwise its always required behavior.

Reply:
Good point, but is it okay to represent false documentation when we know its a bug at first place ?

Reply:
Changes done. Will treat this as a bug in future.